lib/i18n/pl_PL.xlf

```diff
@@ -700,421 +700,425 @@
+własną ścieżkę. Przykł. "Nowiny"</target>
 			</segment>
 		</unit>
 		<unit id="167">
 			<segment>
 				<source>Types</source>
-				<target>Types</target>
+				<target>Typy</target>
 			</segment>
 		</unit>
 		<unit id="168">
 			<segment>
 				<source>Notes</source>
-				<target>Notes</target>
+				<target>Notatki</target>
 			</segment>
 		</unit>
 		<unit id="169">
 			<segment>
 				<source>You can leave a field blank to keep the size free</source>
-				<target>You can leave a field blank to keep the size free</target>
+				<target>Można pozostawić puste pole,
+aby utrzymać swobodny rozmiar</target>
 			</segment>
 		</unit>
 		<unit id="170">
 			<segment>
 				<source>Edit</source>
-				<target>Edit</target>
+				<target>Modyfikuj</target>
 			</segment>
 		</unit>
 		<unit id="171">
 			<segment>
 				<source>Order</source>
-				<target>Order</target>
+				<target>Porządek</target>
 			</segment>
 		</unit>
 		<unit id="172">
 			<segment>
 				<source>Action</source>
-				<target>Action</target>
+				<target>Działanie</target>
 			</segment>
 		</unit>
 		<unit id="173">
 			<segment>
 				<source>Follow URL</source>
-				<target>Follow URL</target>
+				<target>Idź za URL</target>
 			</segment>
 		</unit>
 		<unit id="174">
 			<segment>
 				<source>Open URL (new window)</source>
-				<target>Open URL (new window)</target>
+				<target>Otwórz URL (nowe okno)</target>
 			</segment>
 		</unit>
 		<unit id="175">
 			<segment>
 				<source>Download file</source>
-				<target>Download file</target>
+				<target>Pobierz plik</target>
 			</segment>
 		</unit>
 		<unit id="176">
 			<segment>
 				<source>Watch image</source>
-				<target>Watch image</target>
+				<target>Spójrz na obraz</target>
 			</segment>
 		</unit>
 		<unit id="177">
 			<segment>
 				<source>Website</source>
-				<target>Website</target>
+				<target>Witryna WWW</target>
 			</segment>
 		</unit>
 		<unit id="178">
 			<segment>
 				<source>Services</source>
-				<target>Services</target>
+				<target>Usługi</target>
 			</segment>
 		</unit>
 		<unit id="179">
 			<segment>
 				<source>Thumbnails</source>
-				<target>Thumbnails</target>
+				<target>Miniatury</target>
 			</segment>
 		</unit>
 		<unit id="180">
 			<segment>
 				<source>Item</source>
-				<target>Item</target>
+				<target>Element</target>
 			</segment>
 		</unit>
 		<unit id="181">
 			<segment>
 				<source>Hidden</source>
-				<target>Hidden</target>
+				<target>Ukryty</target>
 			</segment>
 		</unit>
 		<unit id="182">
 			<segment>
 				<source>Rollover image</source>
-				<target>Rollover image</target>
+				<target>Przewiń obraz</target>
 			</segment>
 		</unit>
 		<unit id="183">
 			<segment>
 				<source>Do nothing</source>
-				<target>Do nothing</target>
+				<target>Nic nie rób</target>
 			</segment>
 		</unit>
 		<unit id="184">
 			<segment>
 				<source>Web page</source>
-				<target>Web page</target>
+				<target>Strona WWW</target>
```

Comment:
@stpk007 the translation for this unit feels weird, is possible to have a more specific word instead of WWW? (same in unit 8). These words are used in the menu and represent the specific website the user is editing.